src/styles/blocks/card.scss

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+  font-family: $main-font;
+  color: $main-color;
+  font-weight: 400;
+
+  &:hover {
+    transform: scale(120%);
+    transition-duration: 300ms;
```

Comment:
Transition should be added to the default state of the element, not hover, to work smoothly